cluster/examples/kubernetes/ceph/operator.yaml

```diff
@@ -77,9 +79,17 @@ spec:
     - name: Age
       type: date
       JSONPath: .metadata.creationTimestamp
+    - name: CephImage
+      type: string
+      description: Ceph Image
+      JSONPath: .spec.cephVersion.image
+    - name: CephName
+      type: string
+      description: Ceph Name
+      JSONPath: .status.cephName
     - name: State
       type: string
-      description: Current State
```

Comment:
should this description still be "Current State"?

Reply:
`Current State` seems okay for `description`. 
It looks like so:
```
 ~ kubectl get cephcluster rook-ceph -n rook-ceph
NAME        DATADIRHOSTPATH   MONCOUNT   AGE   CEPHIMAGE       CEPHNAME   STATE
rook-ceph   /var/lib/rook     3          7m    ceph/ceph:v13   mimic      Created
```

Suggestions on what it can be changed to?

Reply:
Agreed, "Current State" is good for the description. Looks like it just accidentally changed to "Ceph Version"